MatrixSDK/VoIP/CallKit/MXCallKitAdapter.m

```diff
@@ -192,7 +192,14 @@ - (void)reportCall:(MXCall *)call connectedAtDate:(nullable NSDate *)date
 
 + (BOOL)callKitAvailable
 {
-    return [NSProcessInfo.processInfo isOperatingSystemAtLeastVersion:(NSOperatingSystemVersion){10,0,0}];
+	if (@available(iOS 10.0, *)) {
+		// CallKit currently illegal in China
+		// https://github.com/vector-im/riot-ios/issues/1941
+
+		return ![NSLocale.currentLocale.countryCode isEqual: @"CN"];
```

Comment:
https://stackoverflow.com/a/51319158 checks also "CHN". We could do the same.

Reply:
![screen shot 2018-10-16 at 11 04 38 am](https://user-images.githubusercontent.com/5855073/47030454-60a89b80-d133-11e8-9217-bc337f532c9e.png)

It seems like they are all two letters as least as far as iOS is concerned.